.mailmap

```diff
@@ -959,6 +962,7 @@ Mihail Tarigradschi <m.tarigradschi@gmail.com>
 Mihir Wadwekar <m.mihirw@gmail.com>
 Mike Boyle <boyle@astro.cornell.edu>
 Mikel Rouco <mikel.mrm@gmail.com>
+Mikhail Remnev <141655736+maremnev@users.noreply.github.com>
```

Comment:
What name and email address would you like to have recorded in the AUTHORS file?

Reply:
The purpose of this file is to map from the name and email address that is recorded in your commits to the name and email address that should be shown in the AUTHORS file.

Reply:
Your commits are like:
```
From 066a5c52d10d758086ebd7739a9b7b1923ad7ec0 Mon Sep 17 00:00:00 2001
From: Mikhail Remnev <141655736+maremnev@users.noreply.github.com>
Date: Thu, 24 Aug 2023 12:27:54 +0300
Subject: [PATCH] Update qft.py
...
```
It is not generally useful to list the no-reply email address in the AUTHORS file though. Do you want to list an alternate email address?

Reply:
Hypothetically the line to add to .mailmap could be:
```
Mikhail Remnev <maremnev@gmail.com> Mikhail Remnev <141655736+maremnev@users.noreply.github.com>
```
if that was your email adress.

It is also possible to configure git to record your email address like that in the commits rather than using the no-reply email adress.

Reply:
Hi! My name is Mikhail Remnev and email adress is maremnev@gmail.com.
I tried to write string `Mikhail Remnev <maremnev@gmail.com> Mikhail Remnev <141655736+maremnev@users.noreply.github.com>` but tests returned error.

Reply:
Okay, I've added that to .mailmap.

Most likely the error was because you needed to rerun the bin/mailmap_check.py script to reorder the lines in the .mailmap file.